### ISSUE TITLE
Special Characters Break Pages

### ISSUE THREAD

**Issue by lostamongtrees** [RolePlayGateway/roleplaygateway#124]
- #62 via @Law

Also breaks locations and makes them inaccessible IC
![image](https://user-images.githubusercontent.com/52435234/60839356-fb05af00-a19a-11e9-9fe5-e50e07801a27.png)

Unable to even be teleported to
![image](https://user-images.githubusercontent.com/52435234/60839420-1cff3180-a19b-11e9-86d6-e58d07f612d5.png)

**Comment by lostamongtrees** [comment]
Character is now trapped - cannot teleport out

![image](https://user-images.githubusercontent.com/52435234/60839833-2046ed00-a19c-11e9-88c9-284ea8374119.png)

**Comment by lostamongtrees** [comment]
Created a character into 
https://www.roleplaygateway.com/universes/the-multiverse/places/der-kalteste-umarmung

Won't load, stuck at .357

![image](https://user-images.githubusercontent.com/52435234/60840696-14f4c100-a19e-11e9-8434-8dfe7f56e1a0.png)

refreshed

![image](https://user-images.githubusercontent.com/52435234/60840716-1f16bf80-a19e-11e9-91c1-0345d85b1976.png)

Aha! If you make the character there you can access it. Went ahead and properly connected it IC so it's at least accessible.

**Comment by lostamongtrees** [comment]
![image](https://user-images.githubusercontent.com/52435234/60859934-527b3d80-a1e2-11e9-9c07-ef745d9c1618.png)

stores, too- think this is the only one
https://www.roleplaygateway.com/roleplay/the-multiverse/places/yosai-fish-market

**Comment by littlekreen** [comment]
This is very likely due to mysql using the gimped version of UTF-8. the database would need to be dumped then all UTF8 text fields migrated to UTF8MB4

**Comment by martindale** [comment]
There's a string formatting function that has to be called for unicode storage, in some places it isn't being used.  We'll have to pare these down one at a time (and can probably do online table upgrades if necessary).

**Comment by lostamongtrees** [comment]
& seems to do it too

![image](https://user-images.githubusercontent.com/52435234/67484359-f542b500-f635-11e9-995f-70ff5989cf59.png)

**Comment by lostamongtrees** [comment]
> Suddenly, this area begins to warp and meltâeverything here begins to twist and turn, and finally evaporate... before taking shape as something completely new:

Are semicolons not translating? It seems wherever there is a semicolon it has formatting issues.
